static/includes/CreateFeatureRequest.md

```diff
@@ -0,0 +1,47 @@
+&NewLine;
+
+Want to see a new feature added to TrueNAS?
+You can see and vote for community-proposed features in the TrueNAS Community Forum [Feature Requests](https://forums.truenas.com/c/features/) category.
+This is the place to suggest improvements and feature functionality to be considered for addition to the TrueNAS development roadmap.
+
+## Submitting and Voting for Requests
+A TrueNAS forums account is required to submit or vote for feature suggestions.
+
+To submit a new feature request, click **<i class="fa fa-plus" aria-hidden="true"></i> New Topic** in the top right of any forum screen
+Enter a title that describes the requested improvement or new functionality.
+Change the category tag to **Feature Requests**.
+The body of the post populates a template.
+Describe the **Problem/Justification**, **Impact**, and provide **User Story** for your request.
+Click **<i class="fa fa-plus" aria-hidden="true"></i> Create Topic** to submit the request.
+
+{{< trueimage src="/images/Contribute/FeatureRequest.png" alt="Submit a Feature Request" id="Submit a Feature Request" >}}
+
+To vote for a Feature Request, open that thread use the **Vote** button in the top left corner, next to the title.
```

Comment:
feature request in this case is not a proper noun, and it is talking about it in a general sense and not the form name, so lower case.
```suggestion
To vote for a feature request, open that thread then click **Vote** at the top left corner, next to the title.
```